### DISCUSSION TITLE
Unable to start pdns service

### DISCUSSION THREAD

**Discussion by yuril123** [PowerDNS/pdns#14305] [Q&A]
Hello, 
I am getting the following error starting pdns service.
pdns_server[26910]: Error parsing bind configuration: Unable to open '/var/named/chroot/etc/named.rfc1912.zones': Permission denied
![image](https://github.com/PowerDNS/pdns/assets/45941536/e6adb530-3af6-4279-802a-6d5b2263c2eb)

I have checked this file "named.rfc1912.zones" and it's  exists  and contain correct data:

zone "mydomain.com" IN {
        type master;
        file "/var/named/chroot/var/named/mydomain.com.zone";
        allow-update { none; };
};

Please help.

**Comment by omoerbeek** [comment]
Look at the file ownership and permissions. The server is not able to read the file.
